library/src/com/anjlab/android/iab/v3/BillingProcessor.java

```diff
@@ -51,6 +54,9 @@
 		void onBillingInitialized();
 	}
 
+	private static final Date dateMerchantLimit1=new Date(2012, 12, 5); //5th December 2012
+	private static final Date dateMerchantLimit2=new Date(2015, 7, 20); //21st July 2015
```

Comment:
1st date is a date when introduced format of orderId=[merchantId].[actualOrderId] [see here](http://stackoverflow.com/questions/16277467/protection-from-freedom)
2nd date is a date when aforementioned format was changed (actually few days ago) - [see here](http://stackoverflow.com/questions/31602234/google-play-order-id-updated-to-new-format)